dynacred/src/Contact.ts

```diff
@@ -481,6 +542,22 @@ export class Contact {
                 case 0:
                     this.credential.setAttribute("1-devices", "initial", this.seedPublic.toBuffer());
                     break;
+                case 1:
+                    const csBufOld = this.credential.getAttribute("1-public", "contacts");
+                    if (csBufOld && csBufOld.length > 0) {
```

Comment:
I think you can replace it with simply `csBufOld !== undefined` but maybe the second check is relevant.

Reply:
as there is a `setAttribute` line afterwards, I guess that the field `1-public` / `contacts` stays in the structure, but I'm not sure whether it will have a `buffer.alloc(0)` or `undefined`. Better safe than sorry.